[Bug]: Build of 2.6.4 F-Droid flavor fails thanks to Crashlytics

### Contact Details

_No response_

### What happened?

The build failed, at `:app:uploadCrashlyticsMappingFileFdroidRelease` –  something that should not even exist for a FOSS flavor. Hence F-Droid will be unable to build –and we at IzzyOnDroid cannot confirm your APK as reproducible.

### App Version

2.6.4

### Phone

n/a

### Device

n/a

### Firmware

n/a

### Relevant log output

```shell
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:uploadCrashlyticsMappingFileFdroidRelease'.
> java.io.IOException: Unknown error while sending file, check network [/build/repo/app/build/crashlytics/fdroidRelease/.crashlytics-mappings-tmp/6a9e954530
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/buildSrc/src/main/kotlin/Configs.kt
+++ b/buildSrc/src/main/kotlin/Configs.kt
@@ -22,6 +22,6 @@ object Configs {
     const val COMPILE_SDK = 36
     const val VERSION_CODE = 30604 // format is Mmmss (where M is 1+the numeric major number
     const val VERSION_NAME = "2.6.4"
-    const val USE_CRASHLYTICS = true // Set to false if you don't want to use Firebase Crashlytics
+    const val USE_CRASHLYTICS = false // Set to true if you want to use Firebase Crashlytics
     const val MIN_DEVICE_VERSION = "2.5.14" // Minimum device firmware version supported by this app
 }

